src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Serialization/MessagePack/Formatters/RazorConfigurationFormatter.cs

```diff
@@ -47,7 +47,7 @@ public override RazorConfiguration Deserialize(ref MessagePackReader reader, Ser
             ? version
             : RazorLanguageVersion.Version_2_1;
 
-        return new(languageVersion, configurationName, extensions, ForceRuntimeCodeGeneration: forceRuntimeCodeGeneration);
+        return new(languageVersion, configurationName, extensions);
```

Comment:
Why are we bothering to read `forceRuntimeCodeGeneration` above if it's not used any longer? It looks like the read was removed from the JSON serialization.

Reply:
There is potential overlap where we have it written to a file right?

Reply:
It's super small though. Is it worth protecting between 17.10p2 and 17.10p3? (Was this even in 17.10p2?)

Reply:
oh actually, it's shipped in vs code already.

Reply:
We need a flow chart!